scout/server/blueprints/variants/templates/variants/indicators.html

```diff
@@ -84,15 +86,12 @@
   {% endif %}
 {% endmacro %}
 
+{% macro evaluations_badge(variant) %}
```

Comment:
I like a good macro, but this is just a chaining - effectively just renaming the macro. Let's call the generic one directly instead!